VelocidiSDK.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>FILEHEADER</key>
+    <string> Created for ___PROJECTNAME___ in ___YEAR___
+// Copyright (c) ___YEAR___ ___ORGANIZATIONNAME___. 
+// Licensed under the Apache License, Version 2.0 (the "License");
+    </string>
```

Comment:
Yes, this is not needed. I added it in the past but during an offline conversation, we decided not to have any fileheaders. I still kept it just because it looks better to have that being automatically inserted then a `//`. I can remove it if you want